client/src/utils/date.js

```diff
@@ -10,6 +10,7 @@ dayjs.extend(isBetween);
 export const dayUtils = dayjs;
 
 export const dateToString = (dateObj) => dayjs(dateObj).format('YYYY/MM/DD');
+export const dateToTextString = (dateObj) => dayjs.utc(dateObj).format('MMM DD, YYYY');
```

Comment:
Not a fan of these names. All strings are text strings... the difference is the formatting. The names of these functions should reflect that.

Reply:
Updated to differentiate between long vs short form. If you have a better name suggestion let me know!

Reply:
Looks good enough to me!